docs/contributing.md

```diff
@@ -216,6 +216,56 @@ to use" instructions](https://github.com/rui314/mold#how-to-use).
 On recent versions of MacOS, the default linker Rust uses is already
 multi-threaded. It should use all the CPU cores without any configuration.
 
+### Editor setup
+
+#### Visual Studio Code
+
+We recommend at least these settings:
+
+```js
```

Comment:
nit: does github support `json` (which i guess this is)? maybe it gets rendered in a nicer way then somehow

Reply:
For this specific block -- we could use `json`.

The next block though is json with comments, where were have several options:

* `json` will look good in the docs but bad on github
* `jsonc` does not work in the docs
* `js` looks a bit worse but is acceptable

If we prioritize docs over github, then I'd use `json`, otherwise `js` as a compromise